src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -25,21 +36,28 @@ mutable struct MAInfo
 end
 
```

Comment:
Change the default constructor to avoid user supplied values that cause problems. This can interfere with testing though, so you may want to provide an override? Or just ignore it?

Reply:
This type does not have a default constructor as this is an organizational structure that we do not aim for the user to have access to.